experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -162,10 +197,6 @@ function Interfacer.update_field!(atmos_sim::ClimaAtmosSimulation, ::Val{:co2},
     end
 end
 # extensions required by the Interfacer
-function Interfacer.update_field!(sim::ClimaAtmosSimulation, ::Val{:surface_temperature}, csf)
-    sim.integrator.p.radiation.radiation_model.surface_temperature .= CA.RRTMGPI.field2array(csf.T_S)
-end
```

Comment:
Do we no longer require atmos models to provide `update_field!` for `surface_temperature`? If so, we should update the interfacer docs

Reply:
Atmos is now using the surface temperature in the surface thermo state to set `radiation_model.surface_temperature` internally. But now you mention it, this update is dependent on atmos calculating turbulent surface fluxes (basically one parameterization depends on the output of the other 😓), so without turbulent fluxes, radiation won't get its update. I.e. we should keep this. I will also comment on this in the code.